cmd/juju/secretbackends/add.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2022 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package secretbackends
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/juju/cmd/v3"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"github.com/juju/utils/v3/keyvalues"
+	"gopkg.in/yaml.v2"
+
+	"github.com/juju/juju/api/client/secretbackends"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/secrets/provider"
+	_ "github.com/juju/juju/secrets/provider/all"
+)
+
+type addSecretBackendCommand struct {
+	modelcmd.ControllerCommandBase
+	out cmd.Output
+
+	AddSecretBackendsAPIFunc func() (AddSecretBackendsAPI, error)
+
+	Name    string
+	Backend string
```

Comment:
For consistency, `BackendType` would be better?
https://github.com/juju/juju/pull/14949/files#diff-f4d1c4834e9a0c8ac391d8157bc4d5f444c04db6f9673853cdbba0f0a8f192f2R13

Reply:
yes, ty